FitVid/src/index.js

```diff
@@ -7,21 +7,27 @@ import { makeServer } from "./server";
 import { ToastProvider } from "./Context/toast-context";
 import { UserProvider } from "./Context/user-context";
 import { LikedProvider } from "./Context/liked-context";
+import { HistoryProvider } from "./Context/history-context";
+import { WatchLaterProvider } from "./Context/watch-later-context";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
-    <LikedProvider>
-      <UserProvider>
-        <ToastProvider>
-          <Router>
-            <App />
-          </Router>
-        </ToastProvider>
-      </UserProvider>
-    </LikedProvider>
+    <WatchLaterProvider>
```

Comment:
You can have one file for all the contexts and import it over here .

Reply:
Will do it